compiler/circledump/src/Dump.cpp

```diff
@@ -369,18 +369,22 @@ void dump_model(std::ostream &os, const circle::Model *model)
   auto opcodes = reader.opcodes();
   auto buffers = reader.buffers();
   auto metadata = reader.metadata();
+  auto signaturedefs = reader.signature_defs();
 
   // dump operator_codes
   os << "Operator Codes: [order] OpCodeName (OpCode Enum)" << std::endl;
   int32_t opcode_index = 0;
   for (auto opcode : opcodes)
   {
     circle::BuiltinOperator op_code = opcode->builtin_code();
+    // cast to int32_t to print as number or int_8 will print as ascii code
```

Comment:
(just for curiosity) Why did you use `int_8`, not `int8_t` or something? I cannot find `int_8` in this repo.

Reply:
`int_8 ` is typo :) it should be `int8_t`. I'll fix it.